test/jinahub/encoder/image/requirements.txt

```
@@ -0,0 +1 @@
+pytest==6.2.4
```

Please use `git+https` installation in the hub so that we won't polutate the pypi statistics.

```suggestion
git+https://github.com/jina-ai/jina.git@v2.0.0rc6#egg=jina[http]
pytest==6.2.4
```